packages/core/schematics/migrations.json

```
@@ -5,15 +5,20 @@
       "description": "Angular v17 introduces a new control flow syntax that uses the @ and } characters. This migration replaces the existing usages with their corresponding HTML entities.",
       "factory": "./migrations/block-template-entities/bundle"
     },
-    "migration-v17-compiler-options": {
+    "compiler-options": {
       "version": "17.0.0",
       "description": "CompilerOption.useJit and CompilerOption.missingTranslation are unused under Ivy. This migration removes their usage",
       "factory": "./migrations/compiler-options/bundle"
     },
-    "migration-transfer-state": {
+    "transfer-state": {
       "version": "17.0.0",
       "description": "Updates `TransferState`, `makeStateKey`, `StateKey` imports from `@angular/platform-browser` to `@angular/core`.",
       "factory": "./migrations/transfer-state/bundle"
+    },
+    "control-flow": {
+      "version": "17.0.0",
+      "description": "Converts the entire application to block control flow syntax",
```

You need to mark this as optional.

```suggestion
      "description": "Converts the entire application to block control flow syntax",
       "optional": true,
```

---

Ah right, good catch.